src/CostApi/AzureCostApiRetriever.cs

```diff
@@ -210,7 +210,7 @@ private async Task<HttpResponseMessage> ExecuteCallToCostApi(bool includeDebugOu
 
             var currency = row[3].ToString();
 
-            var costItem = new CostItem(date, value, valueUsd, currency);
+            var costItem = new CostItem(date, value, valueUsd, currency, "");
```

Comment:
You set the currency to an empty string here. A specific reason for that? This should represent the currency the billing is in.

Reply:
No. Last argument is the Tag. Reason to be empty is because in that method includeTags do not apply

Reply:
Oh sorry, misreading it here.